modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -22,17 +23,19 @@ In earlier versions of {product}, defining conditional policies with a policy ac
 
 This issue led to difficulties in defining conditional policies, hindering the configuration of permissions within the application.
 
-With this fix, you can now define conditional policies in {product-short} with permission policy actions of 'use'. link:{LinkRHIDPIssue}RHIDP-2237[(RHIDP-2237)]
+With this fix, you can now define conditional policies in {product-short} with permission policy actions of 'use'. 
```

Comment:
I get that in the code this is 'use', but if it's a literal name of a field or something, should it be `use` in our docs maybe? I could be very wrong.